software/clipbucket.yml

```diff
@@ -0,0 +1,13 @@
+name: "ClipBucket"
+website_url: "https://clipbucket.oxygenz.fr/en/"
+source_code_url: "https://github.com/MacWarrior/clipbucket-v5"
+description: "Start your own video sharing website (YouTube/Netflix Clone) in a matter of minutes."
+licenses:
+  - AAL
+platforms:
+  - Docker
+  - PHP
+tags:
+  - Media Streaming - Multimedia Streaming
+  - Photo and Video Galleries
+demo_url: "https://demo.clipbucket.oxygenz.fr/"
```

Comment:
It doesn't look like this links to an interactive demo, so it should probably be removed. Other than that looks good.

Reply:
```suggestion
```

Reply:
This demo seems interactive, allowing users to click through the different links? A demo doesn't need to allow for logging in.

Reply:
Yes, the démo offers a full overview of the functionalities.

Reply:
I would argue that this can be added back. I checked both on my phone and laptop and it seems to be interactive enough to warrant be added.